user/views.py

```diff
@@ -1,3 +1,56 @@
-from django.shortcuts import render
+import re
+import json
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models  import User, Language, Country
+from my_settings  import SECRET_KEY, ALGORITHM
+
+REGEX_EMAIL     = '([a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+(\.[a-zA-Z]{2,4}))'
+REGEX_PASSWORD  = '^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$'
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            name     = data['name']
+            email    = data['email']
+            password = data['password']
+            language = Language.objects.get(name='ko')
+            country  = Country.objects.get(name='KR')
+            assert not User.objects.filter(username = name), "ALREADY_EXISTS_ACCOUNT"
+            assert re.match(REGEX_EMAIL, email), "INVALID_EMAIL_FORMAT"
+            assert re.match(REGEX_PASSWORD, password), "INVALID_PASSWORD_FORMAT"
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode()
+            User.objects.create(email = email, password = hashed_password, language = language, country = country, username = name)
```

Comment:
```suggestion
            User.objects.create(
                email = email,
                password = hashed_password,
                language = language,
                country = country,
                username = name
            )
```
속성이 많은데 한 줄에 모두 작성되어 있습니다.
분리해주시는 것이 보다 가독성 높은 코드가 되겠네요 :)
줄 분리 후에는 `=` 기준으로 코드도 정렬해주시는 것을 추천드립니다.

Reply:
`=` 기준으로 정렬되도록 수정했습니다!

Reply:
create할 시 지정하는 속성들을 위 초록색 코드와 같이 개별 줄로 분리하는 수정 사항이  반영되지 않은 것 같아요 :) 다시 한 번 확인해주세요